client/app/src/main/java/com/example/criengine/Activities/AddBookActivity.java

```diff
@@ -0,0 +1,99 @@
+package com.example.criengine.Activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.example.criengine.Database.DatabaseWrapper;
+import com.example.criengine.Objects.Book;
+import com.example.criengine.Objects.Profile;
+import com.example.criengine.R;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.material.snackbar.Snackbar;
+
+public class AddBookActivity extends AppCompatActivity {
+
+    private Profile bookProfile;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_add_book);
+        // Get database for profile
+        final DatabaseWrapper dbw = DatabaseWrapper.getWrapper();
+
+        // Set buttons and warning field to views
+        final Button cancelButton = findViewById(R.id.newBookCancelButton);
+        final Button saveButton = findViewById(R.id.newBookSaveButton);
+
+        // The fields of the book
+        final EditText bookTitle = findViewById(R.id.newBookTitle);
+        final EditText bookDesc = findViewById(R.id.newBookDesc);
+        final EditText bookAuthor = findViewById(R.id.newBookAuthor);
+        final EditText bookISBN = findViewById(R.id.newBookISBN);
+        // TODO: replace editTExt with an actual image
+        final EditText bookImageURL = findViewById(R.id.newBookImageURL);
+        final TextView warning = findViewById(R.id.newBookWarning);
+
+        // Get the current profile
+        dbw.getProfile(dbw.userId).addOnSuccessListener(new OnSuccessListener<Profile>() {
+            @Override
+            public void onSuccess(Profile profile) {
+                bookProfile = profile;
+            }
+        });
+
+        // Save button is clicked
+        saveButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (!bookTitle.getText().toString().isEmpty()
+                        && !bookDesc.getText().toString().isEmpty()
+                        && !bookAuthor.getText().toString().isEmpty()
+                        && !bookISBN.getText().toString().isEmpty()) {
+
+                    // Create the book
+                    Book newBook = new Book(bookProfile.getUserID(),
+                            bookTitle.getText().toString(),
+                            bookAuthor.getText().toString(),
+                            bookDesc.getText().toString(),
+                            bookISBN.getText().toString(),
+                            "Available");
```

Comment:
Oh lol, sorry I just realized that this breaks stuff.
The status according to the requirements is "available" and the other activities which depend on the status are all lower case.
Can you change this to be lower case so we don't need to make other changes <3